recipes/parg/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            bin_path = os.path.join("bin", "test_package")
+            self.run("%s -h" % bin_path, run_environment=True)
```

Comment:
Do we really need to parse? Can we simplify the text package?

Reply:
IMOO, test package is simple enough

Reply:
yes, I am not sure what do you mean, if you can illustrate with some example, would be nice